vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -391,6 +394,11 @@ func applyPodLimitRange(resources []vpa_types.RecommendedContainerResources,
 	for i := range pod.Spec.Containers {
 		limit := (*fieldGetter(resources[i]))[resourceName]
 		cappedContainerRequest, _ := scaleQuantityProportionally(&limit, &sumLimit, &targetTotalLimit)
+		if resourceName == apiv1.ResourceMemory {
+			// Round down
```

Comment:
I couldn't find a better way to round down, if you know one I welcome suggestions :)

Reply:
Wouldn't AsScale work? https://github.com/kubernetes/autoscaler/blob/02d67527d779beed73916362bcb3e334959690e8/vertical-pod-autoscaler/vendor/k8s.io/apimachinery/pkg/api/resource/quantity.go#L502

Reply:
`AsScale` rounds up too

https://github.com/kubernetes/autoscaler/blob/02d67527d779beed73916362bcb3e334959690e8/vertical-pod-autoscaler/vendor/k8s.io/apimachinery/pkg/api/resource/quantity.go#L500

Reply:
I see, that's a bit weird 🤷‍♀️